style.css

```diff
@@ -88,14 +90,15 @@ html, body {
 }
 .header__container {
     display: flex;
+    padding: 0 10px;
 }
 .header__menu {
     position: fixed;
     top: -100%;
     left: 0;
-    width: 100%;
+    width: 100vw;
```

Comment:
What is the difference between 100% and 100vw ?

Reply:
No difference here.
I don't remember why I changed it (may be when I tried changed everything to vw).
Changed it back to %